xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
+        <value>The cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+This property gets the cipher suite that is going to be used in the communication between a client and server by the underlying SSL/TLS implementation. This property is only available after a successful call to <xref:System.Net.Security.SslStream.AuthenticateAsServer%2A> or <xref:System.Net.Security.SslStream.AuthenticateAsClient%2A>.
```

Comment:
When this property is available the channel already uses the selected cipher suite (so `is going to be used` should be re-phrased to something like `being used` or just `used`). Also part about `underlying implementation` is rather confusing. I'd keep it simple to something like:
`Cipher suite negotiated during SSL/TLS handshake. Property is only available after the handshake is complete (after successful call to ...<whatever you already have>)`